manager/integration/tests/common.py

```diff
@@ -2433,3 +2433,24 @@ def wait_for_engine_image_deletion(client, core_api, engine_image_name):
         break
 
     assert deleted
+
+
+def create_snapshot(longhorn_api_client, volume_name):
+    volume = longhorn_api_client.by_id_volume(volume_name)
+
+    snapshots = volume.snapshotList(volume=volume_name)
+    snapshots_count = len(snapshots.data)
+
+    volume.snapshotCreate()
```

Comment:
snapshotCreate() should return the name of the snapshot as a part of response, which can be used to determine if the snapshot has been created or not.

Reply:
fixed